plugins/Toolbox/resources/qml/ToolboxLicenseDialog.qml

```diff
@@ -42,7 +42,7 @@ UM.Dialog
             anchors.right: parent.right
             anchors.topMargin: UM.Theme.getSize("default_margin").height
             readOnly: true
-            text: licenseDialog.licenseContent
+            text: licenseDialog.licenseContent || ""
```

Comment:
I don't think this one is needed as licenseDialog is already set to an empty string if none exists and is therefore not shown. Shouldn't need to check for it again in QML.